apps/worker/src/app/workflow/usecases/send-message/send-message-chat.usecase.ts

```diff
@@ -182,11 +176,11 @@ export class SendMessageChat extends SendMessageBase {
     }
 
     const message: MessageEntity = await this.messageRepository.create({
-      _notificationId: notification._id,
+      _notificationId: command.notificationId,
       _environmentId: command.environmentId,
       _organizationId: command.organizationId,
       _subscriberId: command._subscriberId,
-      _templateId: notification._templateId,
+      _templateId: command.templateId,
```

Comment:
```suggestion
      _templateId: command._templateId,
```

For brewity would suggest to add the "_" for all internal id's we having. This has caused some confusion previously specifically with regard to `subsriberId`